package.json

```diff
@@ -3,7 +3,7 @@
 	"publisher": "aminecharot",
 	"displayName": "Azure Blueprints Generator",
 	"description": "Generates the directory of Azure Blueprints",
-	"version": "0.1.0",
```

Comment:
Since this is a new feature I think we should bump up the minor version number (0.2.0 instead)